File: Acceso-a-Datos-master/EVA2/Colegio_API/Colegio_API/Models/Curso.cs
using System.Text.Json.Serialization;

namespace Colegio_API.Models {
  public class Curso {

    public int ID { get; set; }
    public string NombreCurso { get; set; }
    public string Localizacion { get; set; }
    [JsonIgnore]
    public List<Alumno> Alumnos { get; set; }

  }
}

File: angular/src/classes/curso.ts
export class Curso {

  id: number;
  nombreCurso: string;
  localizacion: string;

  constructor(id: number, nombreCurso: string, localizacion: string) {
    this.id = id;
    this.nombreCurso = nombreCurso;
    this.localizacion = localizacion;
  }
}
